barretenberg/cpp/src/barretenberg/vm/tests/AvmMini_execution.test.cpp

```diff
@@ -60,24 +61,28 @@ TEST_F(AvmMiniExecutionTests, basicAddReturn)
                                "00000000"; // ret size 0
 
     auto bytecode = hex_to_bytes(bytecode_hex);
-    auto instructions = Execution::parse(bytecode);
+    auto instructions = Deserialization::parse(bytecode);
 
     // 2 instructions
     EXPECT_EQ(instructions.size(), 2);
 
     // ADD
     EXPECT_EQ(instructions.at(0).op_code, OpCode::ADD);
-    EXPECT_EQ(instructions.at(0).operands.size(), 3);
-    EXPECT_EQ(instructions.at(0).operands.at(0), 7);
-    EXPECT_EQ(instructions.at(0).operands.at(1), 9);
-    EXPECT_EQ(instructions.at(0).operands.at(2), 1);
-    EXPECT_EQ(instructions.at(0).in_tag, AvmMemoryTag::U8);
+
+    auto operands = instructions.at(0).operands;
```

Comment:
You may be able to replace all these expectations with something like

```
EXPECT_THAT(operands, ElementsAre(static_cast<uint32_t>(7),...));
```

If things get tricky with types LMK.
http://google.github.io/googletest/reference/matchers.html

Reply:
See above comment.